tests/modeladmin/tests.py

```diff
@@ -121,7 +122,11 @@ class BandAdmin(ModelAdmin):
             fields = ["name"]
 
         ma = BandAdmin(Band, self.site)
-        self.assertTrue(ma.lookup_allowed("name__nonexistent", "test_value"))
+        mock_request = MockRequest()
+        self.assertIs(
+            ma.lookup_allowed("name__nonexistent", "test_value", request=mock_request),
```

Comment:
I'd chop `request=` (from all tests):
```suggestion
            ma.lookup_allowed("name__nonexistent", "test_value", mock_request),
```